src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -82,6 +82,49 @@ describe('Schemaform <SaveInProgressIntro>', () => {
     ).to.equal('testing');
     tree.unmount();
   });
+  it('should render in progress message with header', () => {
+    const user = {
+      profile: {
+        savedForms: [
+          {
+            form: VA_FORM_IDS.FORM_10_10EZ,
+            metadata: {
+              lastUpdated: 946684800,
+              expiresAt: moment().unix() + 2000,
+            },
+          },
+        ],
+        prefillsAvailable: [],
+      },
+      login: {
+        currentlyLoggedIn: true,
+        loginUrls: {
+          idme: '/mockLoginUrl',
+        },
+      },
+    };
+
+    const tree = shallow(
+      <SaveInProgressIntro
+        saveInProgress={{ formData: {} }}
+        pageList={pageList}
+        formId="1010ez"
+        user={user}
+        fetchInProgressForm={fetchInProgressForm}
+        removeInProgressForm={removeInProgressForm}
+        toggleLoginModal={toggleLoginModal}
+        formConfig={formConfig}
+        headingLevel={1}
+      />,
+    );
+
+    expect(tree.find('h1.usa-alert-heading').text()).to.contain(
+      'Your application is in progress',
+    );
+    expect(tree.find('h1')).to.exist;
```

Comment:
I don't think we need the second `expect` now that the first one is specifically selecting a `h1`.

Reply:
Removed :-)